packages/css-library/package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "@department-of-veterans-affairs/css-library",
-  "private": true,
+  "version": "0.0.1",
```

Comment:
Would it be an issue to start at this version number because version `0.1.0` already exists and if/when we ever make a `minor` version update to the package, we will not be able to publish it as `0.1.0`.

https://www.npmjs.com/package/@department-of-veterans-affairs/css-library?activeTab=versions

cc @micahchiang

Reply:
hmm that's a good point. Tbh, I was the one that told @it-harrison it'd probably be okay to start at `0.0.1`, so that's my bad. I think given the life cycle stage css library is in, it's probably not too big of a deal to eventually just skip over `0.1.0`, unless folks feel strongly about that.

Reply:
No big deal. Someone might just be a little confused for minute. If we're following [Semantic Versioning](https://semver.org/#spec-item-4), the most important release will be `1.0.0` anyway which should signal it's ready for the public.

- > (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.
- > Version 1.0.0 defines the public API.